pkg/minishift/oc/oc.go

```diff
@@ -226,3 +225,21 @@ func mergeKubeConfigs(configs []*clientcmdapi.Config) *clientcmdapi.Config {
 	}
 	return mergedConfig
 }
+
+func getKubeConfigGlobalPath() (string, error) {
+	kubeConfigGlobalPath := os.Getenv("KUBECONFIG")
+	if kubeConfigGlobalPath != "" {
+		kubeConfigGlobalPath = strings.FieldsFunc(kubeConfigGlobalPath, splitKubeConfig)[0]
```

Comment:
So the idea is to take the first config in the path, right? Might be worth to document, probably in a godoc:
"getKubeConfigGlobalPath returns the path to the first entry in KUBECONFIG or if not set ~/home/.kube/config" - something along these lines. 

This will help other people to understand the code. I don't think it is so well known that KUBECONFIG can have multiple path elements.

Reply:
> So the idea is to take the first config in the path, right? Might be worth to document, probably in a godoc

@hferentschik Sure that something we can add.

> I don't think it is so well known that KUBECONFIG can have multiple path elements.

https://kubernetes.io/docs/tasks/access-application-cluster/configure-access-multiple-clusters/#set-the-kubeconfig-environment-variable